### PR TITLE
Update user test baseline script to diff against a single authoratative WIP master branch

### LINKED ISSUE #37221: User test PRs are hard to parse and work through for individual PRs
Was chatting with @ahejlsberg today and we were commiserating about how difficult it is to understand user test PRs generated from @typescript-bot. (e.g. `user test this`).

To me, the issue is that the workflow just feels bad.

1. I'm told that the user tests _failed_ (always with the italics!)
1. I open up the user tests PR, which is slow because GitHub is slow
1. I open up the files view, which is slow because GitHub is slow
1. The differences are often just the deltas between today and yesterday's baselines.

Some ideas:

1. We only update user tests on a weekly basis. This avoids the noise and means someone doesn't have to update the differences every day.
1. Maybe diffs can be posted on the PR itself, within a `<details>` tag, e.g.

   `````markdown
   <details>
   <summary>
     The user tests <em>failed!</em>.
     Click here to view the diff of the user tests.
   </summary>

   ```diff
     derp
   - blop
   + bloop
   ```
   </details>
   `````

### PR DESCRIPTION
**Pull request by weswigham**
(like RWC)

Fixes #37221, mostly.

@sandersn we probably just need to merge this and try it out, being a build script~

### CODE REVIEW

**Comment by weswigham** [comment]
@sandersn

**Comment by sandersn** [comment]
I am curious what the new behaviour on the CI update thread will be, since I have that thread pinned in outlook.

**Comment by weswigham** [comment]
>I am curious what the new behaviour on the CI update thread will be, since I have that thread pinned in outlook.

Every time a new `master` PR gets made it should still get a ping, which may be less than daily now, since it'll continuously reuse the existing branch.
